### PR TITLE
Remove path wherever possible

### PR DESCRIPTION
**Pull request by Henni**
would be great if this could be tested especially on windows :)

### CODE REVIEW

File: api/controllers/artists.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
-const path = require('path');
-var context = require(path.join(__dirname, '../../index.js')).getContext();
+
+var context = require('../../index.js').getContext();
```
Comment (sacdallago) [inline api/controllers/artists.js:3]: nah nah! Use Path! What's this sorcery?!?!

**Comment by sacdallago** [comment]
Me not like this! use `path` and `__dirname`! If the problem is `..`, https://shapeshed.com/writing-cross-platform-node/

**Comment by kordianbruck** [comment]
require uses __dirname by default: https://stackoverflow.com/questions/13488501/nodejs-readdir-and-require-relative-paths

**Review by kordianbruck — APPROVED** [review]
Fine....

**Pull request merged**